### PR TITLE
[NORMAL] Fix/cc on review page

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1vqjfwc

### CODE REVIEW

File: components/Pages/CarePackages/BrokerCareCharge/ClaimsCollector.js [hunk 1]
```diff
@@ -1,50 +1,42 @@
 import React from 'react';
 import { Controller } from 'react-hook-form';
-import { Container, HorizontalSeparator, FormGroup, RadioGroup, Select, Textarea } from 'components';
+import { Container, FormGroup, HorizontalSeparator, RadioGroup, Select, Textarea } from 'components';
+import { COLLECTING_REASON_OPTIONS } from 'constants/variables';
 
-export const ClaimsCollector = ({ control, errors, collectedBy, isS117Client }) => {
-  const collectingReasonOptions = [
```
Comment (mugnivenko) [inline components/Pages/CarePackages/BrokerCareCharge/ClaimsCollector.js:6]: here i just delete this variable, and use constant instead

File: components/Pages/CarePackages/CareCharge/ProvisionalCareCharge.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { memo } from 'react';
 import { Controller, useWatch } from 'react-hook-form';
 import { currency } from 'constants/strings';
 import { FormGroup, Input, Label, RadioGroup, Select, Textarea } from 'components/HackneyDS';
-import { careChargeAPIKeys, careChargeFormKeys, collectingReasonOptions } from 'constants/variables';
+import { careChargeAPIKeys, careChargeFormKeys, COLLECTING_REASON_OPTIONS } from 'constants/variables';
```
Comment (mugnivenko) [inline components/Pages/CarePackages/CareCharge/ProvisionalCareCharge.js:5]: rename constant to Upper case

File: components/Pages/CarePackages/ReviewPackageDetails/PackageInfo.js [hunk 3]
```diff
@@ -60,10 +64,12 @@ const PackageInfo = ({ fncDetails, headerTitle, items, containerId, careChargeCl
                   <span className="font-weight-bold">FNC assessment been carried out: </span>
                   {fncDetails.assessmentFileUrl}
                 </p>
+                <HorizontalSeparator height={8} />
                 <p>
                   <span className="font-weight-bold">Collected by: </span>
-                  {fncDetails.funcClaimCollector}
+                  {fncDetails.fncClaimCollector}
```
Comment (mugnivenko) [inline components/Pages/CarePackages/ReviewPackageDetails/PackageInfo.js:70]: replace wrong variable name

File: components/Pages/ServiceUser/Packages/PackageRequest.js [hunk 4]
```diff
@@ -60,7 +60,7 @@ const PackageRequest = ({ packageRequest }) => {
         </Button>
       </Container>
       <HorizontalSeparator height="10px" />
-      {packageRequest?.notes && packageRequest.notes.length && (
+      {!!packageRequest?.notes?.length && (
```
Comment (mugnivenko) [inline components/Pages/ServiceUser/Packages/PackageRequest.js:63]: if length === 0 it show 0 in layout, i just convert it to boolean to fix issue

**Comment by mugnivenko** [comment]
image
fix field to careCharges

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
better name for COLLECTING_REASON_OPTIONS

**Comment by mugnivenko** [comment]
> better name for COLLECTING_REASON_OPTIONS

i changed name to CLAIM_REASON_OPTIONS. If this is wrong name, you can suggest your own version

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
merge latest, i need to test this with the latest work to double check something

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
url: /care-package/930ed6cb-8f45-4995-a061-b7b6dd87e7a0/details

Develop:
Screenshot 2021-12-09 at 11 09 12
your branch:
Screenshot 2021-12-09 at 11 09 02

why are you not showing cc information?  we should show this bit **if the care charge collector is supplier:**
Screenshot 2021-12-09 at 11 10 13
but not this bit:
Screenshot 2021-12-09 at 11 10 48

**if collector is hackney** then we show it all like this:
Screenshot 2021-12-09 at 11 11 15

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
view fnc link doesnt work
care-package/4bb38003-8872-4cd8-ad39-12987b227fca/review

Screenshot 2021-12-14 at 13 41 24

**Pull request merged**